portfolio/src/main/webapp/script.js

```diff
@@ -66,8 +66,13 @@ function makeLarger(pic){
  * fetches comments from data servlet
  */
 function getComments(){
-    fetch('/data').then(response => response.json()).then((comments) => {
     const commentsContainer = document.getElementById('comments-container');
+    commentsContainer.innerHTML = "";
+
+    var commentLimit = getCommentLimit();
```

Comment:
Prefer `const` or `let` instead of `var`.
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/let#Description